codegens/nodejs-axios/lib/parseRequest.js

```diff
@@ -0,0 +1,194 @@
+var _ = require('./lodash'),
+  sanitize = require('./util').sanitize;
+  // path = require('path');
+
+
+/**
+ * Parses URLEncoded body from request to axios syntax
+ *
+ * @param {Object} body URLEncoded Body
+ * @param {boolean} trim trim body option
+ * @param {boolean} ES6_enabled ES6 syntax option
+ */
+function parseURLEncodedBody (body, trim, ES6_enabled) {
+  var varDeclare = ES6_enabled ? 'const' : 'var',
+    bodySnippet = varDeclare + ' qs = require(\'qs\')\n',
+    dataArray = [];
+
+  _.forEach(body, function (data) {
+    if (!data.disabled) {
+      dataArray.push(`'${sanitize(data.key, trim)}': '${sanitize(data.value, trim)}'`);
+    }
+  });
+  bodySnippet += varDeclare + ` data = qs.stringify({\n ${dataArray.join(',\n')} \n});`;
+  return bodySnippet;
+}
+
+/**
+ * Parses Formdata from request to axios syntax
+ *
+ * @param {Object} body FormData body
+ * @param {boolean} trim trim body option
+ * @param {boolean} ES6_enabled ES6 syntax option
+ */
+function parseFormData (body, trim, ES6_enabled) {
+  var varDeclare = ES6_enabled ? 'const' : 'var',
+    bodySnippet = varDeclare + ' FormData = require(\'form-data\')\n';
+  bodySnippet += varDeclare + ' data = new FormData();\n';
+  // check if there's file
+  const fileArray = body.filter(function (item) { return !item.disabled && item.type === 'file'; });
+  if (fileArray.length > 0) {
+    bodySnippet += varDeclare + ' fs = require(\'fs\')\n';
```

Comment:
need a semicolon at the end of the require('fs') statement

Reply:
done